extensions/wikia/Oasis/Oasis.i18n.php

```diff
@@ -161,12 +161,19 @@
 your email preferences here: http://community.wikia.com/Special:Preferences',
 	'oasis-label-wiki-vertical-id-0' => 'Other',
 	'oasis-label-wiki-vertical-id-1' => 'TV',
-	'oasis-label-wiki-vertical-id-2' => 'Video Games',
```

Comment:
We're going to leave it as is for now and chat with Noelle about possibly updating other languages at a later date.